cms/static/js/views/settings/grading.js

```diff
@@ -298,10 +303,13 @@ var GradingView = ValidatingView.extend({
         this.descendingCutoffs.push({designation: this.GRADES[gradeLength], cutoff: failBarWidth});
         this.descendingCutoffs[gradeLength - 1]['cutoff'] = Math.round(targetWidth);
 
-        var $newGradeBar = this.gradeCutoffTemplate({ descriptor : this.GRADES[gradeLength],
-            width : targetWidth, removable : true });
+        var newGradeHtml = this.gradeCutoffTemplate({
+            descriptor : this.GRADES[gradeLength],
+            width : targetWidth,
+            contenteditable: true,
+            removable : true }).toString();
         var gradeDom = this.$el.find('.grades');
-        gradeDom.children().last().before($newGradeBar);
+        gradeDom.children().last().before(newGradeHtml.toString());
```

Comment:
We decided to update the linter to allow for the following:
```
gradeDom.children().last().before(HtmlUtils.ensureHtml(newGradeHtml).toString());
```
You should be able to make that update now and it should work.